src/model/TypeHierarchy.ts

```diff
@@ -0,0 +1,64 @@
+import ComplexDataType from "./dataTypes/ComplexDataType";
+
+/**
+ * The type of the internal map used to represent the hierarchy.
+ * The key is {{namespace}}.{{typeName}} of a type. The value is a Set of
+ * its children
+ */
+interface IHierarchyMap {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5648116)